### ISSUE TITLE
Update site header

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#816]
1) Make the logo left-aligned
2) Update the logo (add 'City Of Detroit')
3) Increase the size of the links
4) Modify 'Log In' to 'Sign In'
5) Add 'Sign Up' link to the right

When signed in, 'Sign In' and 'Sign Up' links should be removed and replaced with 'My Account' link (styled the same as the other links).

Design is available in [Figma](https://www.figma.com/file/wAhoXVmHRIJ2TyBKFVq2JY/Desktop-and-Mobile-Experiences?node-id=3540%3A13497) (01.1) 

![site_header](https://user-images.githubusercontent.com/394908/145118695-d03068d1-fa1e-4014-9dd8-6b555b5bf151.png)
